hosting/single/runner.sh

```diff
@@ -45,8 +45,7 @@ fi
 # randomise any unset environment variables
 for ENV_VAR in "${ENV_VARS[@]}"
 do
-    temp=$(eval "echo \$$ENV_VAR")
-    if [[ -z "${temp}" ]]; then
+    if [[ -z "${!ENV_VAR}" ]]; then
```

Comment:
Are we sure this doesn't break backwards compat with existing single image users?

Reply:
Yes, if a user was setting any variable that the image supported, the existing solution didn't respect the value, thus randomly generating one for them and storing it in a file on disk so that it would remain persistent during container restarts.

What this should do is start respecting whatever they were adding as variables. If they weren't using them and defaulting to randomly generated ones, their values would be on disk, and if they're not, they'll still be randomly generated.

TLDR: the change ensures that [what we claim](https://docs.budibase.com/docs/docker#method-1---docker-run) in the documentation holds.